### PR TITLE
 ♻️ Refactor Revoke Interviewer Role endpoint to work with User Id instead of InterviewerId

### PR DESCRIPTION
**Pull request by CalculusEnjoyer**

### CODE REVIEW

**Comment by CalculusEnjoyer** [comment]
|File|Coverage [91.27%]|:green_apple:|
|:-|:-:|:-:|
|[CoordinatorController.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/96bbd1dec25e52065c86e79f012eeab88e32674c/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FCoordinatorController.java)|91.27%|:green_apple:|

|Total Project Coverage|66.15%|:green_apple:|
|:-|:-:|:-:|

**Review by VladOSIO2 — APPROVED** [review]
Approved!

File: src/main/java/com/intellias/intellistart/interviewplanning/controllers/CoordinatorController.java [hunk 1]
```diff
@@ -192,10 +192,11 @@ public ResponseEntity<List<InterviewerDto>> getInterviewers() {
...
   @DeleteMapping("/users/interviewers/{interviewerId}")
   public ResponseEntity<InterviewerDto> revokeInterviewerRole(@PathVariable Long interviewerId) {
-    Interviewer interviewerToDelete = interviewerService.getInterviewerById(interviewerId);
-    User userToDowngrade = interviewerToDelete.getUser();
+    User userToDowngrade = userService.findUserById(interviewerId);
     userToDowngrade.setRole(Role.CANDIDATE);
     userService.registerUser(userToDowngrade);
+    Interviewer interviewerToDelete = interviewerService.getInterviewerByUserId(
+        userToDowngrade.getId());
     Interviewer interviewer = interviewerService.getInterviewerByUserId(userToDowngrade.getId());
```
Comment (VladOSIO2) [inline src/main/java/com/intellias/intellistart/interviewplanning/controllers/CoordinatorController.java:200]: Found this addition after approving, interviewerToDelete is not used anywhere, is this addition needed here?
